cognite/replicator/__main__.py

```diff
@@ -25,6 +25,9 @@
 
 from . import assets, configure_logger, datapoints, events, files, raw, time_series
 
+# import assets, datapoints, events, files, raw, time_series
```

Comment:
Why are these comments added to the file?

Reply:
Good question, I think he has changed the way of the import, changed it